lib/config/presets/util.ts

```diff
@@ -51,7 +51,7 @@ export async function fetchPreset({
   } else {
     jsonContent = await fetch(
       repo,
-      buildFilePath(`${fileName}.json`),
+      buildFilePath(/\.json5?$/.test(fileName) ? fileName : `${fileName}.json`),
```

Comment:
```suggestion
      buildFilePath(regEx(/\.json5?$/).test(fileName) ? fileName : `${fileName}.json`),
```
please use our re2 wrapper.

Reply:
💡  would be nice to lint this (not sure if it exists an ESLint rule for this)